src/components/Battles.ts

```diff
@@ -326,6 +326,30 @@ export class Battles<TGameStartr extends FullScreenPokemon> extends Component<TG
     }
 
     /**
+     * Tests to see if all party Pokemon have fainted.
+     * @returns If a player's party is wiped or not.
+     */
+    public isPartyWiped(): boolean {
+        for (const chosenPokemon of this.gameStarter.itemsHolder.getItem("PokemonInParty")) {
+            if (chosenPokemon.statistics.health.current !== 0) {
+
```

Comment:
The idea behind putting blank lines before `return`s is to logically separate it from what's before it. If it's the only logic line (other than conditionals) in a section, it's fine leaving it.